sdk/eventhub/azure-eventhub/azure/eventhub/_connection_string_parser.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse  # type: ignore
+
+from ._common import DictMixin
+
+
+class EventHubConnectionStringProperties(DictMixin):
+    """
+    Properties of a connection string.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
missing doc string?

Reply:
The SB parser does not have docstring either, so for consistency, would it be better for me to update this and create a separate PR to update the SB parser as well?

Reply:
discussed with @rakshith91 -- allowing no docstring in init since there is a class doc string